plugins/input/tail.md

```diff
@@ -261,6 +261,21 @@ files with dynamic path setting. I will fix this problem in the future.
 Check [this issue](https://github.com/fluent/fluentd/issues/1126).
 
 
+### pos\_file\_compaction\_interval
+
+| type   | default | version |
+|:-------|:--------|:--------|
+| time   | nil     | 1.9.1   |
```

Comment:
@repeatedly what is the next version? 1.9.2 or 1.10.0 ?

Reply:
@repeatedly ?

Reply:
1.9.2

Reply:
Thank you! fixed https://github.com/fluent/fluentd-docs-gitbook/pull/146/files#diff-bf392d7ffee3a4cc8dc25fc615613797R268